sbom/sbom_test.go

```diff
@@ -48,56 +47,36 @@ func testSBOM(t *testing.T, context spec.G, it spec.S) {
 			}
 
 			type artifact struct {
-				ID string `json:"id"`
+				Name     string   `json:"name"`
+				Version  string   `json:"version"`
+				Licenses []string `json:"licenses"`
+				CPEs     []string `json:"cpes"`
+				PURL     string   `json:"purl"`
 			}
 
 			var syftOutput struct {
 				Artifacts []artifact `json:"artifacts"`
+				Source    struct {
+					Type   string `json:"type"`
+					Target string `json:"target"`
+				} `json:"source"`
+				Schema struct {
+					Version string `json:"version"`
```

Comment:
In https://github.com/paketo-buildpacks/packit/pull/271/files, I noticed you added a `Schema.URL` field as well. Should we include that here too for completeness?

Reply:
I don't think the SBOM URL is very useful to assert on, since it's just a pointer to the schema definition. I think asserting on the schema version is useful, though, because we'll probably want to know if syft starts producing a non-backwards-compatible BOM. See [the README](https://github.com/anchore/syft/blob/a18fbacae7bd375a38e6ece836b6c85571deb80f/schema/json/README.md) of the Syft schema directory for an explanation of "SchemaVer", a cousin of SemVer.